docs/run-node/keyring.md

```diff
@@ -78,15 +82,63 @@ $ pass init <GPG_KEY_ID>
 Replace `<GPG_KEY_ID>` with your GPG key ID. You can use your personal GPG key or an alternative
 one you may want to use specifically to encrypt the password store.
 
-## The `test` backend
+### The `test` backend
 
 The `test` backend is a password-less variation of the `file` backend. Keys are stored
 unencrypted on disk. This backend is meant for testing purposes only and **should never be used
 in production environments**.
 
-## The `kwallet` backend
+### The `kwallet` backend
 
 The `kwallet` backend uses `KDE Wallet Manager`, which comes installed by default on the
 GNU/Linux distributions that ships KDE as default desktop environment. Please refer to
 [KWallet Handbook](https://docs.kde.org/stable5/en/kdeutils/kwallet5/index.html) for more
 information.
+
+## Adding keys to the keyring
+
+Applications developed using the Cosmos SDK come with the `keys` subcommand. For the purpose of this tutorial, we're running the `simd` CLI, which is an application built using the Cosmos SDK for testing and educational purposes. For more information, see [`simapp`](https://github.com/cosmos/cosmos-sdk/tree/v0.40.0-rc1/simapp).
+
+::: warning
+Make sure you can build your own binary, and replace `simd` with the name of your binary in the snippets.
+:::
+
+```
+$ simd keys
+Usage:
+  simd keys [command]
+
+Available Commands:
+
+  add         Add an encrypted private key (either newly generated or recovered), encrypt it, and save to disk
+  delete      Delete the given keys
+  export      Export private keys
+  import      Import private keys into the local keybase
+  list        List all keys
+  migrate     Migrate keys from the legacy (db-based) Keybase
+  mnemonic    Compute the bip39 mnemonic for some input entropy
+  parse       Parse address from hex to bech32 and vice versa
+  show        Retrieve key information by name or address
+
+Flags:
+  -h, --help                     help for keys
+      --keyring-backend string   Select keyring's backend (os|file|test) (default "os")
+      --keyring-dir string       The client Keyring directory; if omitted, the default 'home' directory will be used
+      --output string            Output format (text|json) (default "text")
+
+Global Flags:
+      --home string   directory for config and data (default "/Users/{your_username}/.simapp")
+      --trace         print out full stack trace on errors
+
+Use "simd keys [command] --help" for more information about a command.
```

Comment:
do we want to copy the whole help output? This will mean that every time we change it we will need to remember to update this part. If we want to do it then I would prefer to have an appropriate note in the code to also update the docs.
Otherwise this should be enough:
Use `simd keys` for help about the `keys` command and `simd keys [command] --help` for more information.

Reply:
Plus, we should really instruct users on how to use our autocompletion command:

```
simd completion
```

At the start of a new **bash** session, users can just run:

```
. <(simd completion)
```